flink-dist/src/main/flink-bin/yarn-bin/yarn-session.sh

```diff
@@ -52,5 +52,5 @@ log_setting="-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4
 
 export FLINK_CONF_DIR
 
-$JAVA_RUN $JVM_ARGS -classpath $CC_CLASSPATH:$HADOOP_CLASSPATH:$HADOOP_CONF_DIR:$YARN_CONF_DIR  $log_setting org.apache.flink.yarn.cli.FlinkYarnSessionCli -j $FLINK_LIB_DIR/flink-dist*.jar "$@"
```

Comment:
Thanks. Good catch. 
I changed the script in my installed 1.0.3 version and then committed that.
I missed the change in the classname.
As you can see I just fixed it.